### ISSUE TITLE
The attributes field not returning everything 

### ISSUE THREAD

**Issue by robertodias180** [malcommac/SwiftRichString#56]
I have this style:
```
let inputStyle = Style {
        $0.font = UIFont.bsFont(type: .medium, size: 13)
        $0.color = UIColor.white
        $0.kerning = Kerning.point(0.2)
}
```

On V2.0.1 `inputStyle.attributes` returns 3 atrributes.
On V2.0.5 `inputStyle.attributes` returns 1 atrributes.

**Comment by armcast** [comment]
+ 1

**Comment by malcommac** [comment]
Fixed in 2.1.0.
Now it returns

```
▿ 3 elements
  ▿ 0 : 2 elements
    ▿ key : NSAttributedStringKey
      - _rawValue : NSFont
    - value : <UICTFont: 0x7f90c7d05e40> font-family: "Helvetica"; font-weight: normal; font-style: normal; font-size: 34.00pt
  ▿ 1 : 2 elements
    ▿ key : NSAttributedStringKey
      - _rawValue : NSKern
    - value : 0.2
  ▿ 2 : 2 elements
    ▿ key : NSAttributedStringKey
      - _rawValue : NSColor
    - value : UIExtendedGrayColorSpace 1 1
```
